src/node_errors.cc

```diff
@@ -0,0 +1,19 @@
+#include "node_errors.h"
+#include "node_internals.h"
+#include "v8.h"
+
+using v8::HandleScope;
+
+namespace node {
+namespace errors {
+
+TryCatch::~TryCatch() {
+  if (HasCaught() && mode_ == kFatal) {
+    HandleScope scope(env_->isolate());
```

Comment:
Since this is new code, could you use full names, i.e. `v8::HandleScope` && `node::ReportException` etc.